01_financial-well-being/historical_comparison_income_metric_county.qmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "Historical Income Metric Comparison"
+author: "JP Walsh"
+date: today
+format:
+  html:
+    toc: true
+    toc_float: true
+    embed-resources: true
+    code-fold: true
+execute: 
+  warning: false
+editor_options: 
+  chunk_output_type: console
+---
+
+This program compares the results for income metric updated by JP Walsh in 2023/24 with the results from the metric calculations originally written by Kevin Werner in 2022 for the creation of the UMF income predictor. These processes should be the same and as a result the final metrics should be comparable.
+
+## House keeping
+
+```{r}
+library(tidyverse)
+library(tidylog)
+library(here)
+library(assertr)
+library(gt)
+
+options(scipen = 999)
+```
+
+## Longitudinal Data        
+
+Ready income metric.
+```{r}
+
+counties_old <- read_csv(here("data", "01_mobility-metrics_race-ethnicity_longitudinal.csv")) %>% 
+  select(year, state, county, pctl_income_20_old = pctl_income_20, pctl_income_50_old = pctl_income_50, pctl_income_80_old = pctl_income_80,
+         subgroup) %>% 
+  filter(year == 2021, subgroup == "All")
+
+counties_new <- read_csv(here("01_financial-well-being", "data", "final", "metrics_income_county_race-ethnicity_longitudinal.csv")) %>% 
+  select(year, state, county, pctl_income_20, pctl_income_50, pctl_income_80, subgroup) %>% 
+  filter(year == 2021, subgroup == "All")
+  
+```
+
+Summarize the statistics. Remove NAs from the old dataset where we have added suppression so that the comparison groups are equal.
```

Comment:
Just wondering whether the following discrepancies are acceptable- counties_old has 3100 counties compared with 3143 in counties_new. counties_new has more NA values as well. Quite similar distributions though.